kolibri/core/assets/src/core-app/mediator.js

```diff
@@ -450,13 +450,27 @@ export default class Mediator {
       } else {
         // We have a content renderer for this, but it has not been loaded, so load it, and then
         // resolve the promise when it has been loaded.
-        Promise.all(this._contentRendererUrls[kolibriModuleName].map(this._scriptLoader))
-          .then(() => {
+        const urls = this._contentRendererUrls[kolibriModuleName].filter(
+          url =>
+            (languageDirection === languageDirections.RTL &&
+              url.includes(languageDirections.RTL)) ||
+            !url.includes('css')
```

Comment:
Are URLs literally HTTP URL strings, and it's doing string searches for things like `css` and the RTL constants in them? If so, this seems potentially brittle and could match unintended files.

Reply:
Yes, and yes it could. Not quite sure why I didn't at least use `endsWith`.

Reply:
maybe add a comment describing the expected form of the URLs. (currently not clear what they look like)